### ISSUE TITLE
client.puased() error

### ISSUE THREAD

**Issue by oih741** [Streamedian/html5_rtsp_player#150]
HI.
I am a programmer at Suntech, a Korean company. name is InHyeok Oh.
We are testing now before purchasing a license.
We keep getting this error but don't know the cause.Is there a solution?
There are two questions.

1. Uncaught TypeError: Cannot read property 'paused' of undefined
    at WSPlayer.isPlaying (streamedian.min.js:9560)
    at HTMLVideoElement.<anonymous> (streamedian.min.js:9513)

-> I guess it won't run after stopping because of this error.

2. When the rtsp is received in real time, buffering occurs after the sink reaches the end.
Is it possible to have autorun when the buffer fills up more than a certain amount?
Right now buffering is taking place.

![image](https://user-images.githubusercontent.com/22514655/69936235-8866e880-151a-11ea-9af7-86670dd33e6d.png)

Thank you for reading this article.

**Comment by abhik261738** [comment]
i have the same issue can you please tell me how to solve this?

**Comment by MakdDevelop** [comment]
Hello @oih741 
I find two players in a screenshot. Does the pause error occur when using a single player?
Could you share your page, so that we can check the error?

**Comment by abhik261738** [comment]
Yes while using single player, i am getting the error the screenshot is attached bellow,

Sometime paused is working sometime it's throwing the error,

I am running the player in my localhost,

When refreshing the error will go and run as usual.

![Workspace 1_006](https://user-images.githubusercontent.com/52736098/73151750-8ec8a980-40f2-11ea-94d6-caf2d47f1330.png)
![Workspace 1_007](https://user-images.githubusercontent.com/52736098/73151751-8ec8a980-40f2-11ea-8f3e-f8ab1b4c0473.png)
